app/poros/food.rb

```diff
@@ -0,0 +1,9 @@
+class Food
+  attr_reader :code, :description, :ingredients, :brand
+  def initialize(data)
+    @code = data[:foodCode]
+    @description = data[:description]
+    @ingredients = data[:foodNutrients].map {|nutrient| nutrient[:nutrientName]}
```

Comment:
We were actually expecting you to use the ingredients key in the same hash as these other attributes but this is an interesting approach! Especially when the ingredients key has an empty value